README.md

```diff
@@ -1,6 +1,7 @@
 # saltstack-netapi-client-java [![Build Status](https://travis-ci.org/SUSE/saltstack-netapi-client-java.svg?branch=master)](https://travis-ci.org/SUSE/saltstack-netapi-client-java)
 
-Java bindings for the [SaltStack API] (http://docs.saltstack.com/en/latest/ref/netapi/all/salt.netapi.rest_cherrypy.html#module-salt.netapi.rest_cherrypy.app), please have a look at the [Javadoc] (http://suse.github.io/saltstack-netapi-client-java/docs/v0.1.0).
+Java bindings for the [SaltStack API] (http://docs.saltstack.com/en/latest/ref/netapi/all/salt.netapi.rest_cherrypy.html#module-salt.netapi.rest_cherrypy.app), please have a look at the latest [Javadoc] (http://suse.github.io/saltstack-netapi-client-java/docs/master).
+Javadocs for a specific version can be found at `http://suse.github.io/saltstack-netapi-client-java/docs/<version>`.
```

Comment:
I would suggest not to put this generic link here, simply because you cannot click it. Instead we could maybe just have one link to the latest release and one to the current master docs? I know, we will forget to update the link when we make a release, but listing all versions' links would be a bit too much I guess.